docker/generic/start_proxy.py

```diff
@@ -135,6 +135,28 @@ def make_argparser():
         manually. Valid schemes are `http`, `https`, `grpc`, and `grpcs`.
         '''.format(backend=DEFAULT_BACKEND))
 
+    parser.add_argument(
+        '--enable_backend_address_override',
```

Comment:
Please update the help text for --backend flag too, either refer it to here, copy the override part over

Reply:
Done